warehouse/csp.py

```diff
@@ -86,6 +86,7 @@ def includeme(config):
                 "connect-src": [
                     SELF,
                     "https://api.github.com/repos/",
+                    "https://api.github.com/search/issues",
```

Comment:
Is it correct to not include a trailing "/"?
When the URL to be called is on this form `https://api.github.com/search/issues?q=repo:pypi/warehouse+type:issue+state:open&per_page=1`

Reply:
Yes, my understanding is that the CSP rules are evaluated as direct string matches, and only allow wildcards `*` - so this should be fine.